templates/volunteers/attendance.html

```diff
@@ -33,7 +33,7 @@ <h3 class="text-center">Volunteers Attendance</h3>
                         {% for vol_att in today_vol_att %}
                         <tr>
                             <td>{{ vol_att.volun.roll_no }}</td>
-                            <td>{{ vol_att.volun.profile.get_full_name }}</td>
+                            <td><a href="{% url 'volunteers:profile' vol_att.volun.profile.user_id%}">{{ vol_att.volun.profile.get_full_name }}</a></td>
```

Comment:
Can you format it better? Put each element in a separate line.

Reply:
ok, I will do that.